server/auth/handlers/handle_profile.go

```diff
@@ -137,7 +140,8 @@ func (h *AuthHandlers) profileProc(req *request.AuthReq) error {
 		return nil
 	}
 
-	avatarUrl := fmt.Sprintf("%sapi/system/attachment/avatar/%d/original/%s", GetLinks().Base, u.Meta.AvatarID, types.AttachmentKindAvatar)
+	fullpathAPI := "/" + strings.TrimPrefix(options.CleanBase(options.HttpServer().BaseUrl, options.HttpServer().ApiBaseUrl), "/")
```

Comment:
duplicate as above, create a private method for this.
also, you can put all the vars in one multi-line Sprintf no?

Reply:
This can also be an anonymous function; pick your poison